soroban-preflight/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = "soroban-preflight"
```

Comment:
Can we rename the crate to `soroban-simulation`? Preflight is a specific use case. Simulation is a broad capability that we'll likely use in more places that preflight. I believe we also discussed the vocab we'd use for this and settled on simulation, which is also what we use in APIs such as soroban-rpc.

Reply:
@2opremio I've reserved the `soroban-simulation` crate on crates.io for us to use: https://crates.io/crates/soroban-simulation

Reply:
> Can we rename the crate to `soroban-simulation`? Preflight is a specific use case. Simulation is a broad capability that we'll likely use in more places that preflight. I believe we also discussed the vocab we'd use for this and settled on simulation, which is also what we use in APIs such as soroban-rpc.

Done!